packages/react/src/blocks/Symbol.tsx

```diff
@@ -160,6 +160,9 @@ class SymbolComponent extends React.Component<PropsWithChildren<SymbolProps>> {
                     ...(!this.isEditingThisSymbol && {
                       key: builderComponentKey,
                       noEditorUpdates: true,
+                      query: {
```

Comment:
this query param should not be condition in editingThisSymbol. it should always be provided

Reply:
also, this query param should only be added when the entry value exists AND when the apiEndpoint value is "content"